From 24da05e8fe62fcddc7970155b41e9efcdd28a43b Mon Sep 17 00:00:00 2001
From: Avery Pennarun <apenwarr@gmail.com>
Date: Fri, 24 Jan 2014 03:09:49 -0500
Subject: [PATCH 6/7] mwifiex: remove custom regulatory domain stuff.

Just use the cfg80211 regulatory domain, which in our build, we know we've
configured correctly for our region.
---
 drivers/net/wireless/mwifiex/cfg80211.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/wireless/mwifiex/cfg80211.c b/drivers/net/wireless/mwifiex/cfg80211.c
index 21ee27a..19d3ed1 100644
--- a/drivers/net/wireless/mwifiex/cfg80211.c
+++ b/drivers/net/wireless/mwifiex/cfg80211.c
@@ -2917,11 +2917,13 @@ int mwifiex_register_cfg80211(struct mwifiex_adapter *adapter)
 		wiphy->flags |= WIPHY_FLAG_SUPPORTS_TDLS |
 				WIPHY_FLAG_TDLS_EXTERNAL_SETUP;
 
+#if WANT_CUSTOM_REG
 	wiphy->regulatory_flags |=
 			REGULATORY_CUSTOM_REG |
 			REGULATORY_STRICT_REG;
 
 	wiphy_apply_custom_regulatory(wiphy, &mwifiex_world_regdom_custom);
+#endif
 
 #ifdef CONFIG_PM
 	wiphy->wowlan = &mwifiex_wowlan_support;
-- 
1.9.1.423.g4596e3a

